vocabularies/EntityRelationship.xml

```diff
@@ -193,19 +193,25 @@ An Entity Type can have multiple IDs:
 
       <EnumType Name="temporalIntervalTypeEnum">
         <Member Name="CLOSED_CLOSED">
+          <Annotation Term="Core.Description" String="Closed, Closed interval including the start and end property." />
```

Comment:
"start and end value" or similar (property value, boundary, ...), in all four descriptions